### PR TITLE
Tc quotes

### PR DESCRIPTION
**Pull request by SoapGentoo**

### CODE REVIEW

File: app-admin/ryzen_smu/ryzen_smu-0.1.2.ebuild [hunk 1]
```diff
@@ -26,7 +26,7 @@ pkg_setup() {
 	linux-mod_pkg_setup
 
 	BUILD_TARGETS="modules"
-	BUILD_PARAMS="CC=$(tc-getCC) KERNEL_BUILD=${KERNEL_DIR}"
+	BUILD_PARAMS="CC="$(tc-getCC)" KERNEL_BUILD=${KERNEL_DIR}"
```
Comment (SoapGentoo) [inline app-admin/ryzen_smu/ryzen_smu-0.1.2.ebuild:29]: wrong

File: app-cdr/dvdisaster/dvdisaster-0.79.5.ebuild [hunk 2]
```diff
@@ -37,7 +37,7 @@ DOCS=( CHANGELOG CREDITS.en README README.MODIFYING TODO TRANSLATION.HOWTO )
 
 src_prepare() {
 	default
-	sed -i -e "s@CC=gcc@CC=$(tc-getCC)@" scripts/bash-based-configure || die
+	sed -i -e "s@CC=gcc@CC="$(tc-getCC)"@" scripts/bash-based-configure || die
```
Comment (SoapGentoo) [inline app-cdr/dvdisaster/dvdisaster-0.79.5.ebuild:40]: wrong

Comment (thesamesam) [inline app-cdr/dvdisaster/dvdisaster-0.79.5.ebuild:40]: check it works fine w/ the double quotes?

File: app-crypt/tc-play/tc-play-2.0-r1.ebuild [hunk 3]
```diff
@@ -40,8 +40,8 @@ pkg_setup() {
 	EXTRA_MAKE_FLAGS=(
 		PBKDF_BACKEND="${backend}"
 		WARNFLAGS=""
-		CC=$(tc-getCC)
-		AR=$(tc-getAR)
+		CC="$(tc-getCC)"
+		AR="$(tc-getAR)"
 		PREFIX=/usr \
```
Comment (thesamesam) [inline app-crypt/tc-play/tc-play-2.0-r1.ebuild:45]:
```suggestion
		PREFIX=/usr
```

File: app-misc/glimpse/glimpse-4.18.6-r1.ebuild [hunk 4]
```diff
@@ -22,7 +22,7 @@ PATCHES=( "${FILESDIR}/${PN}-4.18.6-makefile.patch" )
 
 src_prepare() {
 	sed -i \
-		-e "s:CC=.*:CC=$(tc-getCC):" \
+		-e "s:CC=.*:CC="$(tc-getCC)":" \
```
Comment (SoapGentoo) [inline app-misc/glimpse/glimpse-4.18.6-r1.ebuild:25]: wrong

File: app-shells/heirloom-sh/heirloom-sh-050706-r1.ebuild [hunk 5]
```diff
@@ -27,7 +27,7 @@ src_compile() {
 		"CPPFLAGS=${CPPFLAGS}" \
 		"LDFLAGS=${LDFLAGS}" \
 		"LARGEF=" \
-		"CC=$(tc-getCC)"
+		"CC="$(tc-getCC)""
```
Comment (SoapGentoo) [inline app-shells/heirloom-sh/heirloom-sh-050706-r1.ebuild:30]: wrong

File: app-misc/glimpse/glimpse-4.18.6-r1.ebuild (hunk 4 above)
Comment (thesamesam) [inline app-misc/glimpse/glimpse-4.18.6-r1.ebuild:25]: check

**Review by thesamesam — APPROVED** [review]

**Pull request closed without merging**